history-of-the-quakers-v3/original/19-book-5-chapter-4.adoc

```diff
@@ -128,6 +134,8 @@ he may receive a better crown amongst the blessed.
 
 Which is the prayer of, etc.
```

Comment:
So this looks weird just left as is, but I wasn't really sure how to make it look good, as [sscc] and [sss] didn't feel right to me so much.

Reply:
I think it should probably be a `[.signed-section-closing]` since it seems like we would mark it as that if it had a signature after it.  So let's just make it that, even if it looks a little funny.